images/arm-ubuntu-22-image.md

```diff
@@ -1,9 +1,9 @@
 # Ubuntu 22.04 by Arm Limited 
 
-The Ubuntu 22.04 LTS image with tools is created from the [GitHub Actions Runner Images](https://github.com/actions/runner-images) repository. 
+The Ubuntu 22.04 LTS Arm image is created from the [GitHub Actions Runner Images](https://github.com/actions/runner-images) repository.
```

Comment:
`The Ubuntu 22.04 LTS image` is not this image, it'd be GitHub's image. In order to use `The`, a thing has to be unique/unambiguous.

There are various ways to address this, but it seemed more useful to include `Arm` (since that's the image this text is trying to reference) and drop `with tools` (no one is offering an `Ubuntu 22.04 LTS Arm image` **without tools**).